setup_ubuntu20.04.sh

```diff
@@ -25,7 +38,11 @@ case $answer in
     fi
 
     cd $SCRIPT_DIR/ansible
-    ansible-playbook -i localhost, $SCRIPT_DIR/ansible/localhost-setup-ubuntu20.04-devpc.yml -i $SCRIPT_DIR/inventories/local-dev.ini -e AUTOWARE_DIR=$SCRIPT_DIR --ask-become-pass
+    if [ $noninteractive -eq 1 ]; then
+      ansible-playbook -i localhost, $SCRIPT_DIR/ansible/localhost-setup-ubuntu20.04-devpc.yml -i $SCRIPT_DIR/inventories/local-dev.ini -e AUTOWARE_DIR=$SCRIPT_DIR --extra-vars "yn_gpu=y"
+    else
+      ansible-playbook -i localhost, $SCRIPT_DIR/ansible/localhost-setup-ubuntu20.04-devpc.yml -i $SCRIPT_DIR/inventories/local-dev.ini -e AUTOWARE_DIR=$SCRIPT_DIR --ask-become-pass
```

Comment:
@TakaHoribe out of curiosity: what is this become password used for that I had to type in?

Reply:
@fred-apex-ai With the `--ask-become-pass` arg, the ansible asks you a password interactively to run the ansible commands as a superuser.